sdcm/nemesis.py

```diff
@@ -1965,9 +1966,9 @@ def choose_partitions_for_delete(self, partitions_amount, ks_cf, with_clustering
                     continue
 
                 # Get the max cl value in the partition.
-                cmd = f"select ck from {ks_cf} where pk={partition_key} order by ck desc limit 1"
+                cmd = f"select ck from {ks_cf} where pk={partition_key} order by ck desc"
                 try:
-                    result = session.execute(cmd, timeout=300)
```

Comment:
why are you dropping the timeout ?

Reply:
i think it's not needed in this case. but it can either be left there as is, since not have any bad impact

Reply:
fixed